docs/reference/cat/allocation.asciidoc

```diff
@@ -40,6 +40,40 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=cat-s]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=cat-v]
 
+[[cat-allocation-api-response-body]]
+==== {api-response-body-title}
+
+`shards`::
+Number of primary and replica shards assigned to the node.
```

Comment:
I intentionally omitted the data type here. cat APIs aren't intended for application consumption. Let me know if there's a use case I may be overlooking.

Reply:
:+1: